### PR TITLE
Fix output type for compressed BCF

### PR DESCRIPTION
**Pull request by Takadonet**
Change format for compressed BCF to its corresponding extension of 'bcf_bgzip' Otherwise, you cannot output a compressed BCF at all.

### CODE REVIEW

File: tools/bcftools/macros.xml [hunk 1]
```diff
@@ -254,7 +254,7 @@ $vcfs_list_file
   <xml name="macro_vcf_output">
       <data name="output_file" format="vcf">
         <change_format>
-          <when input="output_type" value="b" format="bcf" />
+          <when input="output_type" value="b" format="bcf_bgzip" />
```
Comment (nsoranzo) [inline tools/bcftools/macros.xml]: In Galaxy 17.09 `bcf_bgzip` is just an alias for `bcf`, for details see https://github.com/galaxyproject/galaxy/pull/4477

Comment (Takadonet) [reply tools/bcftools/macros.xml]: and we are currently stuck on 17.05

Comment (bgruening) [reply tools/bcftools/macros.xml]: @Takadonet is this needed or can you use this with a local patch or update to 17.09?

Comment (Takadonet) [reply tools/bcftools/macros.xml]: I will update to 17.09

Comment (bgruening) [reply tools/bcftools/macros.xml]: Ok, thanks. Closing this PR now.

**Pull request closed without merging**